src/chrome/background/message-handler.ts

```diff
@@ -66,6 +70,30 @@ export const BackgroundMessageHandler =
         }))
       }
 
+      case messageDiscriminator.walletToLedger:
+        return createOrFocusTab(config.popup.pages.ledger)
+          .map((tab) => {
```

Comment:
change to `andThen` you are returning a ResultAsync through `sendMessageWithConfirmation`